stix_shifter_modules/guardium/stix_translation/json/report_from_stix_map.json

```diff
@@ -0,0 +1,46 @@
+{
+"x-ibm-finding": {
+    "fields": {
+      "finding_type": ["datacategory"],
+      "start": ["QUERY_FROM_DATE"],
+      "end": ["QUERY_TO_DATE"],
+      "database_name": ["Database"],
+      "db_type":["db_type"],
+      "Severity":["Severity"]
```

Comment:
Same as the other from-STIX map, you'll want to map the Guardium Severity property to `x-guardium:severity`

Reply:
done